Sources/Avatar.swift

```diff
@@ -24,28 +24,93 @@
 
 import Foundation
 
-public struct Avatar {
-
-    public let image: UIImage?
-
-    public let highlightedImage: UIImage?
-
-    public let placeholderImage: UIImage
-
-    public init(image: UIImage? = nil, highlightedImage: UIImage? = nil, placeholderImage: UIImage) {
-        self.image = image
-        self.highlightedImage = highlightedImage
-        self.placeholderImage = placeholderImage
+open class Avatar: UIView {
```

Comment:
Do you think `AvatarView` would be a better name? Allowing people to identify its a `UIView` subclass?